app/src/main/res/values/strings.xml

```diff
@@ -21,38 +21,38 @@
     <string name="main_configure_sync_btn">Configure Sync Preferences</string>
     <string name="main_configure_misc_btn">Configure Miscellaneous</string>
     <string name="main_sync_now">Sync Now</string>
-    <string name="main_sync_in_progress">Sync in progress...</string>
+    <string name="main_sync_in_progress">Sync in progress…</string>
 
     <!-- NAVIGATION DRAWER ACTIONS //-->
     <string name="nav_sync_all">Sync All</string>
     <string name="nav_configure_calendars">Configure Calendars</string>
     <string name="nav_configure_sync">Configure Sync</string>
-    <string name="nav_sync_logs">Sync Logs</string>
+    <string name="nav_sync_logs" comment="(Verb) Synchronize the logs">Sync Logs</string>
```

Comment:
This is actually exactly the wrong meaning – it's the noun. Clicking on this navigation bar entry shows those logs.